source/grid/grid_in.cc

```diff
@@ -1451,14 +1452,73 @@ GridIn<dim, spacedim>::read_msh(std::istream &in)
           in >> line;
         }
 
-      // if the next block is of kind $Entities, ignore it
+      // if the next block is of kind $Entities, parse it
       if (line == "$Entities")
         {
-          do
+          unsigned long numPoints, numCurves, numSurfaces, numVolumes,
+            numPhysicals;
+          int    tag, physicalTag;
+          double boxMinX, boxMinY, boxMinZ, boxMaxX, boxMaxY, boxMaxZ;
+
+          in >> numPoints >> numCurves >> numSurfaces >> numVolumes;
```

Comment:
It was already like this, but could you convert these to `snake_case` (e.g., `n_points` instead of `numPoints`)?